### PR TITLE
Added new attribute for inter-packet gap(IPG)

### PR DESCRIPTION
**Pull request by prgeor**
New attribute to make Inter-Packget Gap (IPG) configurable for ethernet ports.

### CODE REVIEW

File: inc/saiport.h [hunk 1]
```diff
@@ -1955,6 +1955,16 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_DUAL_MEDIA,
 
+    /**
+     * @brief IPG is mandated by IEEE 802.3 to ensure reliable communication between two ethernet devices.
+     * The minimum recommended inter-packet gap(IPG) is 96 bits (9.6usec for 10GBASE-T and 0.96usec for 100GBASE-T)
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 96 (bits)
```
Comment (kcudnik) [inline inc/saiport.h:1964]: remove "bits" string, only raw number is supported

Comment (prgeor) [reply inc/saiport.h]: removed

File: inc/saiport.h [hunk 2]
```diff
@@ -1955,6 +1955,19 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_DUAL_MEDIA,
 
+    /**
+     * @brief IPG
```
Comment (kcudnik) [inline inc/saiport.h:1959]: maybe some couple words in brief information? not just 3 letters?

**Comment by lguohan** [comment]
can you resolve the conflict the address the comments?

File: inc/saiport.h (hunk 2 above)
Comment (prgeor) [reply inc/saiport.h]: added brief

**Comment by prgeor** [comment]
> can you resolve the conflict the address the comments?

yes, resolved.

**Comment by kcudnik** [comment]
please update aspell and fix warnings

**Comment by prgeor** [comment]
> please update aspell and fix warnings

yes, now fixed

**Review by kcudnik — APPROVED** [review]

**Pull request merged**